<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrafficStatistics</class>
 <widget class="QWidget" name="TrafficStatistics">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>268</width>
    <height>372</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0" colspan="4">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>4324 号公路状态监控:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QComboBox" name="wayComboBox"/>
   </item>
   <item row="1" column="0" rowspan="2" colspan="4">
    <widget class="QLabel" name="tollLabel">
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QComboBox" name="segComboBox"/>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="dirComboBox"/>
   </item>
   <item row="4" column="0" colspan="4">
    <widget class="QLabel" name="statisticsLabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
